projects/core/styles/theme/variables.less

```diff
@@ -1,12 +1,20 @@
 @font-fallback: -apple-system, 'BlinkMacSystemFont', system-ui, 'Roboto', 'Segoe UI', 'Helvetica Neue', sans-serif;
 
 :root {
-    // Deprecated
-    --tui-heading-font: 'Manrope', @font-fallback;
-    --tui-text-font: 'Manrope', @font-fallback;
```

Comment:
These two were in the initial release, let's keep them as deprecated to not introduce a breaking change